external-crates/move/crates/move-package-alt/src/graph/package_info.rs

```diff
@@ -36,6 +36,13 @@ impl<F: MoveFlavor> PackageGraph<F> {
     pub(crate) fn package_info(&self, node: NodeIndex) -> PackageInfo<F> {
         PackageInfo { graph: self, node }
     }
+
+    #[cfg(test)]
```

Comment:
This should eventually be not `#[cfg(test)]` I think, but right now it gives a warning. Maybe it should just be `pub`?

Reply:
Also, it needs a doc